corehq/apps/reports/templates/reports/form/partials/single_form_tree.html

```diff
@@ -7,6 +7,12 @@
                     <strong>{% trans "Hidden Values (Form Calculations)" %}</strong>
                 </td>
             </tr>
+        {% elif 'commcare_usercase/' in question.value %}
+            <tr class="info">
+                <td colspan="2">
+                    <strong>{% trans "User Case Data" %}</strong>
+                </td>
+            </tr>
```

Comment:
hmm. that implies to me that something, somewhere is stripping the normal case from the form rendering, and I'm wondering whether we want to use that same mechanism here (an interesting side effect might be that we also improve the ux of the advanced app builder case blocks too).

Let me know if you'd like more guidance than that and I can poke around and try and figure out how that's happening.